src/profile-logic/profile-data.js

```diff
@@ -113,6 +113,8 @@ export function getCallNodeInfo(
 
     // The callNodeTable components.
     const prefix: Array<IndexIntoCallNodeTable> = [];
+    const firstChild: Array<IndexIntoCallNodeTable> = [];
+    const nextSibling: Array<IndexIntoCallNodeTable> = [];
```

Comment:
Because we don't know the length upfront. Typed arrays don't have a `push` method, you need to set their size explicitly. And I haven't really looked at "resizable buffer" typed arrays, it's possible that we could use those. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#behavior_when_viewing_a_resizable_buffer